Tools/autotest/common.py

```diff
@@ -1301,26 +1302,27 @@ def wait_distance(self, distance, accuracy=5, timeout=30):
                                           % (delta, distance))
         raise WaitDistanceTimeout("Failed to attain distance %u" % distance)
 
-    def wait_servo_channel_value(self, channel, value, timeout=2):
-        """wait for channel to hit value"""
+    def wait_servo_channel_value(self, channel, value, timeout=2, comparator=operator.eq):
+        """wait for channel to exceed value"""
         channel_field = "servo%u_raw" % channel
+        opstring = ("%s"%comparator)[-3:-1]
```

Comment:
I'm *sure* flake8 is going to complain about this line because of the lack of whitespace around `%` :-)

`flake8 common.py` will lint this file for you.  I've no doubt unrelated complaints will pop up at the moment on common.py